pkg/query-service/app/traces/v4/query_builder.go

```diff
@@ -292,12 +292,22 @@ func buildTracesQuery(start, end, step int64, mq *v3.BuilderQuery, panelType v3.
 	if mq.AggregateOperator == v3.AggregateOperatorNoOp {
 		var query string
 		if panelType == v3.PanelTypeTrace {
-			withSubQuery := fmt.Sprintf(constants.TracesExplorerViewSQLSelectWithSubQuery, constants.SIGNOZ_TRACE_DBNAME, constants.SIGNOZ_SPAN_INDEX_V3_LOCAL_TABLENAME, timeFilter)
-			afterSubQuery := tracesV3.AddLimitToQuery(constants.TracesExplorerViewSQLSelectAfterSubQuery, mq.Limit)
-			if mq.Offset != 0 {
-				afterSubQuery = tracesV3.AddOffsetToQuery(afterSubQuery, mq.Offset)
+			if mq.TraceOrdering == v3.TraceOrderingByTraceDuration || mq.TraceOrdering == "" {
```

Comment:
Any reason for not utilizing the existing order by ?

Reply:
I was under the impression, that the existing orderBy is only used for time series and table view, therefore I did not want to insert my filters there as this TraceOrdering is only for the traces Tab. 
Nonetheless, I have edited the PR to include the orderBy field